lib/kernel-manager.coffee

```diff
@@ -164,7 +170,7 @@ class KernelManager
                     detail: (_.map @_kernelSpecs, 'spec.display_name').join('\n')
                 atom.notifications.addInfo message, options
 
-            callback? err, @_kernelSpecs
+            callback? null, @_kernelSpecs
```

Comment:
I don't understand this.

Shouldn't `updateKernelSpecs` fail if `err` in `@getKernelSpecsFromJupyter` and `@getKernelSpecsFromSettings` is empty?